src/main/java/frc/robot/commands/AutoChargeStationSequence.java

```diff
@@ -9,14 +9,13 @@
 import com.pathplanner.lib.PathPlannerTrajectory;
 import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
 import frc.robot.Cal;
-import frc.robot.Constants;
 import frc.robot.subsystems.drive.DriveSubsystem;
 
 /** Drives onto the charge station and balances */
 public class AutoChargeStationSequence extends SequentialCommandGroup {
   private PathPlannerTrajectory traj =
       PathPlanner.loadPath(
-          Constants.PLACEHOLDER_STRING,
+          "EngageOnlyTraj.path",
```

Comment:
I was looking at the examples and I think this doesn't need the file extension, see here: https://github.com/mjansen4857/pathplanner/wiki/PathPlannerLib:-Java-Usage